src/store-update.ios.ts

```diff
@@ -26,6 +23,11 @@ export class StoreUpdate {
       ...config,
       onConfirmed: StoreUpdate._openStore.bind(StoreUpdate),
     })
+
+    //Hook into resume event to check for version
+    app.on(app.resumeEvent, function (args) {
+        StoreUpdate.checkForUpdate();
+    });
```

Comment:
Is the `resumeEvent` triggered at launch time ? If not I would implement `launchEvent ` as well